ur_description/CMakeLists.txt

```
@@ -6,6 +6,9 @@ find_package(ament_cmake REQUIRED)
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
 install(DIRECTORY urdf DESTINATION share/${PROJECT_NAME})
 install(DIRECTORY meshes DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY config DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY cfg DESTINATION share/${PROJECT_NAME})
```

As commented above. I propose to rename this into `rviz`. BTW, a very unusual style for install commands, but it seems UR-maintainers like it :D

```suggestion
install(DIRECTORY rviz DESTINATION share/${PROJECT_NAME})
```